Objects/bytesobject.c

```diff
@@ -3558,3 +3560,148 @@ _PyBytes_Repeat(char* dest, Py_ssize_t len_dest,
     }
 }
 
+/** Dedent a UTF-8 encoded string.
```

Comment:
For me, this function is unrelated to bytes, please move it to Modules/main.c.

Reply:
Please read my comment.
https://github.com/python/cpython/pull/103998#issuecomment-1646670688

I want to add `.dedent()` to bytes, bytearray, and str.

Reply:
Oh, interesting.